.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,33 @@
+# Prerequisites

```

Comment:
Modify the issue templates as
+ Which endpoint you are willing to add?
+ Reference formula for the calculation.
+ Name of the endpoint you want to suggest.